### ISSUE TITLE
transparent png logo display as black square

### ISSUE THREAD

**Issue by Nigrini** [pH7Software/pH7-Social-Dating-CMS#521]
When loading any image that does not have transparency it display normally, but the logo that I ultimately wants to use only display as a black square.

I tested this with other images that is transparent and it does the same for them

**Comment by TGHepburn** [comment]
Have you tried making the logo with transparency in PNG and/or GIF?  I'm sorry I'm not too familiar with this CMS yet but it's my first thought.   Looking right now, the Zendate theme has the default logo in PNG so I'd use PNG format first.

**Comment by polynamaude** [comment]
@TGHepburn Maybe if you give us a screenshot then we'd be better to help you out.
But one thing, you may not be at the right place if you are looking for help regarding HTML5/CSS coding.

I suggest you look at the following
https://www.w3schools.com/html/
https://stackoverflow.com/

This is for issue related to the pH7CMS, not general question about web design. This is related to the use of template.

**Comment by Nigrini** [comment]
To change the logo I logged into the Admin panel -> General settings -> Logo

I have tried any image that is transparent that I have on my local PC, and just for good measure downloaded an image from google images but all of them display a solid black square
![logo](https://user-images.githubusercontent.com/43742116/56612564-fa865f00-6614-11e9-82c1-3291db98057a.PNG)

**Comment by pH-7** [comment]
This issue looks similar to https://github.com/pH7Software/pH7-Social-Dating-CMS/issues/417 (although, wasn't not solved).

**Comment by TGHepburn** [comment]
I am seeing same issue with clean install using "Base" theme

**Comment by polynamaude** [comment]
@TGHepburn  Hi,
Have tried using the developper tools in Firefox or Chrome, more specifically the "inspector" to get the information about your style sheet. You'll see all the levels with attributes.
Again, this is not related to the CMS but more to templating and CSS/HTML5.

**Comment by TGHepburn** [comment]
@polynamaude yes, I use Firefox Web Dev tools.  The logo that gets uploaded is resized (feature) and placed - when using 'Base' theme - here: `templates/themes/base/img` and saved as `logo.png`  This is correct I hope.  When downloaded and view _independent_ of browser it has black where transparency was set in original.  I think this is what @Nigrini sees too.

**Comment by Nigrini** [comment]
Yes - I have given up on this issue and as a bypass use and image with the same colour background as the page

**Comment by TGHepburn** [comment]
@Nigrini If you have access to the file directory of the CMS through FTP or Cpanel, I'd suggest uploading an image with the **correct size** (by looking at image there now) and overwriting the file `logo.png` with one that has your transparent background. If you try this, please report back the results. I haven't tried yet.
